### PR TITLE
📝 Iterate quickstart for better readability

### PR DESCRIPTION
**Pull request by falexwolf**
See title.

### CODE REVIEW

**Comment by ghost** [comment]
<details>
<summary><b>👇 <a href="https://app.codesee.io/r/reviews?pr=71&src=https%3A%2F%2Fgithub.com%2Flaminlabs%2Fbionty">Click on the image</a> for a new way to code review</b>
</summary>

  - Make big changes easier — review code in small groups of related files
  - Know where to start — see the whole change at a glance
  - Take a code tour — explore the change with an interactive tour
  - Make comments and review — all fully sync’ed with github

    [Try it now!](https://app.codesee.io/r/reviews?pr=71&src=https%3A%2F%2Fgithub.com%2Flaminlabs%2Fbionty)
</details>

[Review these changes using an interactive CodeSee Map](https://app.codesee.io/r/reviews?pr=71&src=https%3A%2F%2Fgithub.com%2Flaminlabs%2Fbionty)

#### Legend
CodeSee Map Legend

File: docs/tutorial/quickstart.ipynb [hunk 1]
```diff
@@ -67,9 +69,45 @@
    "source": [
     "Now you have access to a couple of attributes, let's see what they are.\n",
     "\n",
-    "- `.fields`: this gives you a list of column names that annotate each entry, in this case each species\n",
-    "- `.std_id`: this prints the field which is used as the standardized id. Note that Bionty focuses on interpretability, so we use the unique `common_name` of [Ensembl Species](https://asia.ensembl.org/info/about/species.html).\n",
-    "- `.std_name`: this is the value of `std_id` in the current instance. (e.g. `human`, `mouse`, `dog` ...)"
+    "```{Important}\n",
+    "\n",
+    "These are attributes of the base class `bt.Ontology`. So, you'll find the same API for any other entity (such as Gene, Protein, etc.), not just for Species.\n",
```
Comment (falexwolf) [inline docs/tutorial/quickstart.ipynb:71]: Not 100% sure if accurate, but I assume so!

File: docs/tutorial/quickstart.ipynb [hunk 2]
```diff
@@ -67,9 +69,45 @@
    "source": [
     "Now you have access to a couple of attributes, let's see what they are.\n",
     "\n",
-    "- `.fields`: this gives you a list of column names that annotate each entry, in this case each species\n",
-    "- `.std_id`: this prints the field which is used as the standardized id. Note that Bionty focuses on interpretability, so we use the unique `common_name` of [Ensembl Species](https://asia.ensembl.org/info/about/species.html).\n",
-    "- `.std_name`: this is the value of `std_id` in the current instance. (e.g. `human`, `mouse`, `dog` ...)"
```
Comment (falexwolf) [inline docs/tutorial/quickstart.ipynb:72]: This whole list of things is too much information for the reader. So, I spread it out across several calls.

Comment (falexwolf) [reply docs/tutorial/quickstart.ipynb:72]: See what follows below!

File: docs/tutorial/quickstart.ipynb [hunk 3]
```diff
@@ -78,17 +116,34 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "print(\n",
```
Comment (falexwolf) [inline docs/tutorial/quickstart.ipynb:81]: I think these `print()` statements with backticks are incredibly dense & technical and therefore hard to parse for the reader. I replaced them with walking the reader through the API in a staged, 1-by-1 fashion.

Comment (falexwolf) [reply docs/tutorial/quickstart.ipynb:81]: Trying to mimic how the user would actually use the API themselves.

File: docs/tutorial/quickstart.ipynb [hunk 4]
```diff
@@ -78,17 +116,34 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For entities that require name conversions, there is a `.search` method to allow converting among different ids.\n",
+    "\n",
+    "```{admonition} Alex\n",
+    "I think a good name for the method would be \"lookup\". I just don't fully understand why `scientific_name` is not an attribute of `species`. 🤔\n",
+    "```"
```
Comment (falexwolf) [inline docs/tutorial/quickstart.ipynb:134]: OK, so here you see that the intuitive high-level user API is indeed the static API.

The thing I'd want first and foremost is to have `scientific_name` at the tip of my hand and an auto-lookup, without having to call a `.search` method or whatever it'd be called.
